operations/iuf/stages/deliver_product.md

```diff
@@ -0,0 +1,42 @@
+# `deliver-product`
+
+The `deliver-product` stage delivers the content provided in the product distribution file to the system. Artifacts will be uploaded to the following locations based on entries specified in the product `iuf-product-manifest.yaml` file.
+
+| Location                      | `iuf-product-manifest.yaml` Entry |
+| ----------------------------- | --------------------------------- |
+| S3 Content                    | `s3`                              |
+| S3 Loftsman Manifests         | `loftsman`                        |
+| Nexus Docker Registries       | `docker`                          |
+| Nexus Helm Chart Repositories | `helm`                            |
+| Nexus RPM Blob Stores         | `nexus_blob_stores`               |
+| Nexus RPM Repositories        | `nexus_repositories`              |
+| Nexus RPMs                    | `rpms`                            |
+| VCS Content                   | `vcs`                             |
+| IMS Images and Recipes        | `ims`                             |
+
+The versioned product catalog entry for the product is updated as the new content is uploaded.
+
+## Impact
+
+The `deliver-product` stage does not change the running state of the system as the new content has been uploaded but has not been deployed.
```

Comment:
...as the new content has been uploaded, but not deployed.

Reply:
fixed